homeassistant/components/miraie_ac/config_flow.py

```diff
@@ -44,6 +53,12 @@ async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str,
         return {"title": "MirAIe"}
 
 
+def _validate_mobile_number(mobile_number):
+    pattern = r"^\+91\d{10}$"
```

Comment:
+91? is that the country code of india?

Reply:
Yes, I think these devices are only available in India at present